app/models/tv_season.rb

```diff
@@ -25,11 +23,12 @@ def self.parse_results(json)
       end
       @overview = result[:overview]
       @still_path = result[:still_path]
+      # TODO: refactor this into a TvEpisode object
       @episode = TVSeason.new(@episode_number, @name, @air_date, @guest_stars, @overview, @still_path)
```

Comment:
Doing a deeper dive into refactoring `TVSeason` vs `TVEpisode` vs `TVEpisodes` is out of scope for this PR. I've opened PR #252 to address it.